public/locales/de/common.json

```diff
@@ -0,0 +1,18 @@
+{
+  "nav.buy-13th-age-label": "13th Age kaufen",
+  "nav.compendium-label": "Kompendium",
+  "nav.compendium-basic-rules-label": "Basis-Regeln",
+  "nav.compendium-ancestries-label": "Völker",
+  "nav.compendium-classes-label": "Klassen",
+  "nav.guides-label": "Leitfäden",
+  "nav.encounter-builder-label": "Begegnungs-Planer",
+  "nav.close-menu-label": "Menü schließen",
+  "nav.open-menu-label": "Menü öffnen",
+  "not-found.title-label": "Du hast dich verirrt",
```

Comment:
So with the `not-found.title-label`, I made the "404" page of 13 Vaults to be a reference to the Dark Souls game's "YOU DIED" screen.

Here's 13 Vaults:

![image](https://user-images.githubusercontent.com/1108160/221974775-df60c2e3-f035-4c8d-97c6-f64827cc7916.png)

Here's Dark Souls:

![image](https://user-images.githubusercontent.com/1108160/221974890-88b2e537-85e8-4d3f-afca-31f940349e37.png)

The reason why I chose the words "You Got Lost" was because I needed something short to give the same feel to "You Died". With all of that said, is there a way to get this same meaning across (the user is visiting a page that doesn't exist) while also being very short?